lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -98,17 +101,24 @@ class AddInspectionBookingBody extends StatelessWidget {
                             )),
                       ],
                     ),
-                    BookedItemCard(
-                      propertyName: 'Green living space in Melbourne',
-                      propertyType: HatSpaceStrings.current.apartment,
-                      propertyImage:
-                          'https://m.media-amazon.com/images/I/81YR16yC2QL._AC_UF350,350_QL80_.jpg',
-                      price: 4800,
-                      state: 'Victoria',
-                      currency: Currency.aud,
-                      rentingPeriod: 'pw',
-                      onPressed: () {
-                        // TODO: implement BL
+                    BlocBuilder<PropertyDetailCubit, PropertyDetailState>(
+                      builder: (context, state) {
+                        if (state is PropertyDetailLoaded) {
+                          return BookedItemCard(
+                            propertyName: state.name,
+                            propertyType: state.type,
+                            propertyImage: state.photos.first,
+                            price: state.price.rentPrice,
+                            state: state.state,
+                            currency: state.price.currency,
+                            rentingPeriod: ' pw',
```

Comment:
I think we have enum and string for this

Reply:
@suesitran I found out that in the UI does not required rentingPeriod, the `pw` value should be extracted from property's price. Also from our code, we did not have any logic to let user select that as well as store those `pw` value to firebase.
I will consider it as a bug and we should update it from UI and logic from AddProperty Flow

Reply:
Please use localization here

Reply:
already apply localisation